lms/views/dashboard/api/user.py

```diff
@@ -123,7 +123,6 @@ def students_metrics(self) -> APIStudents:
         """Fetch the stats for one particular assignment."""
         assignment = self.dashboard_service.get_request_assignment(self.request)
 
-        request_h_userids = self.request.parsed_params.get("h_userids")
```

Comment:
Unrelated to this PR, but my IDE was warning that this line was unused, because it appears again a few lines later.

See https://github.com/hypothesis/lms/blob/89995361813162f1bdd743905e9f96b1c56a0da9/lms/views/dashboard/api/user.py#L126-L141